### PR TITLE
Lexorin is now better countered with Epinephrine

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
balance: the nanotransen research science division found that epinephrine has healing effects to people who drank lexorin
/:cl:

lexorin is aids, has the highest dps and also stuns to death, if you dont pentetic it away as soon as you get syringed you ll die 100% and even if you pentetic and salbut'd instantly you are likely to get stunned by breathloss for 1 min

now epipen has a chance to save you from certain death, by removing 3u each tick, and 2u of itself and add 1u histamin

i kept its damage and gave it an easy counter because else its gonna just be another useless poison, now you roll your chances if the dude doesnt have an epipen or use a combo of chems instead of straight lexorin (god's intended)

### CODE REVIEW

**Comment by ShibaInuLord** [comment]
Everyone starts with an epipen, this just nerfs lexorin into uselessness.

**Comment by Tlaltecuhtli** [comment]
but not everyone is gonna use it or have it when they really need it

**Comment by Doosteroni** [comment]
Not gonna lie lexorin is the only good poison left as far as I know. I do like this nerf though.

**Comment by Doosteroni** [comment]
15u of lexorin is basically death so yeah I like this

**Comment by wesoda25** [comment]
Epi was always the hard counter to lex so this is just restoring that

**Comment by bluejack1** [comment]
lexorin has already many counters. epipen already slows you from dying, salbutamol too, multiver saves you, so does EVERY PURGING CHEM. "15u Is LeThAl" yes because it's a FUCKING POISON. 
it isnt the only good poison as heparin, sulfonal, chloral, many botany chems, many traitor chems, bath salts and many other mixes will all kill you in one syringe. 
tbh I dont see a good reason to make yet another chem complete shit because "it kills you". any gun or weapon can kill you faster, by the time you're down with lex you should already be chugging a med and screaming over the radio, and of course getting the fuck out of there, so it's not overpowered either. it can already be countered, there are stronger weapons, and it's time consuming to make after the first salbupens and plasma bars run out. If you wanna shit on it because you got killed at least dont have it be countered by a chem that everyone spawns with.

**Comment by evilcyborgmrt** [comment]
Hi I'm a chemist main who rages against every single chem nerf, and I approve of this change. Counterplay good. This will encourage smarter plays as the evil chemist. It is still very easy to murder people with chems.

**Comment by bluejack1** [comment]
> Hi I'm a chemist main who rages against every single chem nerf, and I approve of this change. Counterplay good. This will encourage smarter plays as the evil chemist. It is still very easy to murder people with chems.

stabbing yourself with an epipen isnt counterplay. you start with it. just go ahead and directly remove lexorin at this point

**Comment by Ratvar** [comment]
Yeah, I would like if lexorin and rest of poison chems were removed until someone finds a way to make any of them balanced

**Comment by ATH1909** [comment]
make convermol counter it instead

**Comment by bluejack1** [comment]
> Yeah, I would like if lexorin and rest of poison chems were removed until someone finds a way to make

poisons arent instakills. they kill you slowly. why, in gods name, WHY would you nerf chemistry even more, when the syringe gun has one of the slowest moving and easiest to avoid shots in the game, one ammo roundstart, and even if you get hit you have time to run away or fight back. when most guns will kill you in less than 10 seconds. what is it with the "fuck chem" mindset. just GIVE US ONE BUFF PLEASE

**Comment by Angustmeta** [comment]
1u histamine is pretty pointless due to liver. 20% RNG chance of 5u might work for flavor.

**Comment by Doosteroni** [comment]
Jeez this one guy getting pissy about lexorin nerf is hilarious. Lexorin isn’t honestly THAT hard to make and a basic syringe full is far too fast acting

**Comment by bluejack1** [comment]
> Jeez this one guy getting pissy about lexorin nerf is hilarious. Lexorin isn’t honestly THAT hard to make and a basic syringe full is far too fast acting

I just dont like everyone attacking chems constantly with pointless nerfs. making lexorin by using the salbutamol medipens roundstart still saps medbay of the med. so you'll probably have to make up for it anyway by making more salbutamol. for fuck's sake just give chem one good buff at least. 
they nerfed healing and explosives already, and are trying to nerf clf3 and lex, while adding useless lavaland chems that dont really make up for it.

**Comment by FinnDority** [comment]
Imagine getting annoyed over a poison that only does oxyloss - just ask someone to CPR you.
In an instance where you can't ask for help for one of the easiest-to-deal-with toxins, (aka muted through chems/comms are down/you've been sleeped) you'd probably be dead from just about anything else, assuming you're dealing with anyone even slightly robust. Lexorin, as a purely oxyloss toxin, deals no lasting damage, and if dealt with properly, can easily be ignored as you hunt down a traitor. Giving epinephrine, something everyone spawns with, a way to ignore this makes Lexorin nearly useless - not to mention that epipen refillers are things that exist, and will be nearly immediately constructed and laid down in medical lobbies, removing this "roll your chances" schtick and replacing it with "congratulations, this toxin is completely and utterly useless".

**Comment by 2whatever2** [comment]
I like this pr but why make it the most easiest thing to get in medical and something you start with at least make it convermol or salbutamol because those actually don't start in your roundstart backpack making it epi makes even nuke ops safe against this. Not to mention do you even know how slow epi metabolises? by the time lexorin has been purged there will still be epi and i'm not even mentioning the people who drink epi bottles/pills to make sure they always recover from crit.   I always drink epi and keep it in me in all times due to how it slow it metabolises and how useful it is when you are in lowcrit.

**Comment by OmegaDarkPotato** [comment]
Imagine getting annoyed over a poison that only does oxyloss - just ask someone to CPR you. In an instance where you can't ask for help for one of the easiest-to-deal-with toxins, (aka muted through chems/comms are down/you've been sleeped) you'd probably be dead from just about anything else, assuming you're dealing with anyone even slightly robust. 

Lexorin, as a purely oxyloss toxin, deals no lasting damage, and if dealt with properly, can easily be ignored as you hunt down a traitor. Giving epinephrine, something everyone spawns with, a way to ignore this makes Lexorin nearly useless - not to mention that epipen refillers are things that exist, and will be nearly immediately constructed and laid down in medical lobbies, removing this "roll your chances" schtick and replacing it with "congratulations, this toxin is completely and utterly useless".

**Comment by Tlaltecuhtli** [comment]
it deletes the epipen so you just have to syringe twice

**Comment by necromanceranne** [comment]
I like it adding histamine that's pretty clever for making sure it's not a perfect cure.

**Comment by ATH1909** [comment]
> I like it adding histamine that's pretty clever for making sure it's not a perfect cure.

except your liver pretty much instantly purges the 1u doses of histamine anyway

**Comment by Fox-McCloud** [comment]
Random fact: Lexorin was never intended to be a poison, but a medical chem.

Originally it stopped the breathing proc, but had a 33% chance to cause brute damage. Somewhere along the line, people on Bay/TG got confused as to the functionality and assumed it was a poison, so they altered it to cause oxyloss, which made it super deadly (Oxyloss + no recovery...ouch).

IMO, it's more interesting as a medical chem, provided that it's super hard to create--though it should probably have a harsher trade off like doing 1 tox every cycle or something like that.

**Comment by ATH1909** [comment]
> Random fact: Lexorin was never intended to be a poison, but a medical chem.
> 
> Originally it stopped the breathing proc, but had a 33% chance to cause brute damage. Somewhere along the line, people on Bay/TG got confused as to the functionality and assumed it was a poison, so they altered it to cause oxyloss, which made it super deadly (Oxyloss + no recovery...ouch).
> 
> IMO, it's more interesting as a medical chem, provided that it's super hard to create--though it should probably have a harsher trade off like doing 1 tox every cycle or something like that.

why would such a niche (proposed) medicine chem need a drawback like that? if someone's choking, jab them with oxy meds and drag them out of the room with bad atmos that they're in.

if anything, I'd suggest adding that idea to zombie powder and/or ghoul powder, since survivng without breathing for a while would definitely fit the theme of those chems

**Comment by tmtmtl30** [comment]
the 10u of epinephrine in the medipen would already stop lexorin from being lethal if you just upped its oxy healing to 1 or something. or maybe just make it so that epinephrine only heals 1 oxy damage while the target is at or above 50 oxy damage (knockout threshold). it'd even make sense -- i think a lot of people who don't know the exact functions of chems assume that epinephrine will help people who are at or above 50 oxy damage, even though it doesn't at all. seriously, you're telling me that injecting a fucker full of *adrenaline* doesn't make them wake up faster? come on!

or, you could stop lexorin from dealing any oxy damage (or maybe even adding losebreath stacks) once the target is in crit / oxygen damage exceeds 100. this would basically guarantee that an epinephrine injection would counter a lexorin injection, as even 15u of lexorin will metabolize out of the system far faster than 10u of epinephrine injected at the same time (32 cycles vs. 100 cycles). this would also incentivize more creative mixes than just 15u of lexorin, since now there's no incentive to just stack on lexorin to get more oxygen damage except to lengthen the period they're in crit if they injected themselves with a chem, but that's of course only going to be useful if you're in melee range or can get to them quickly, which isn't necessarily guaranteed.

or! coming back to the 50 oxy damage knockout threshold, you could make it so that epinephrine in somebody's system stops them from passing out due to >50 oxy damage. that doubles the amount of time you have to run away or medicate yourself or cry out for help if you can inject yourself in time, and mean that chemists would have to choose between harder-to-counter chloral-based knockout deterrence and long-term lethality. again, this change would make intuitive sense, since epinephrine = adrenaline and being unconscious from high oxy damage looks a lot like hardcrit.

or you could mix and match the above to get whatever dynamic you think feels right. i think these are all more interesting from a gameplay perspective than just an RNG chance to add histamine, partially because they're about adding mechanics that can be used in counterplay instead of explicitly adding snowflake counterplay purging.

**Comment by Tlaltecuhtli** [comment]
what do you mean

**Comment by tmtmtl30** [comment]
"what do you mean" by *what?* i said a lot of things; i don't want to rewrite my entire goddamn screed if you only don't understand a part

**Comment by Tlaltecuhtli** [comment]
@ExcessiveUseOfCobblestone

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -710,6 +710,11 @@
 	..()
 
 /datum/reagent/medicine/epinephrine/on_mob_life(mob/living/carbon/M)
+	if(holder.has_reagent(/datum/reagent/toxin/lexorin))
+		holder.remove_reagent(/datum/reagent/toxin/lexorin, 3)
+		holder.remove_reagent(/datum/reagent/medicine/epinephrine, 2)
+		if(prob(20))
+			holder.add_reagent(/datum/reagent/toxin/histamine, 4)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:717]: bring 728 up and early return if you have lexorin in your system so if you counter lexorin this way it opens you up to other issues on top of the histamine. Also make it TRUE, it's a boo.

I'd like for it to be less steep in purging both, 3u on iteration is 20% of a syringe gun. Just making it 1 (and not even purging epi) would still make it a good counter but not a stomp for either lexorin users or receivers.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 2]
```diff
@@ -710,6 +710,13 @@
 	..()
 
 /datum/reagent/medicine/epinephrine/on_mob_life(mob/living/carbon/M)
+	if(holder.has_reagent(/datum/reagent/toxin/lexorin))
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:713]: just have `. = TRUE` above here and delete the `. = 1`s in this proc

**Pull request merged**